readme.md

```diff
@@ -0,0 +1,226 @@
+# remark for Visual Studio Code
+
+[![Build][build-badge]][build]
+[![Coverage][coverage-badge]][coverage]
+[![Downloads][downloads-badge]][downloads]
+[![Sponsors][sponsors-badge]][collective]
+[![Backers][backers-badge]][collective]
+[![Chat][chat-badge]][chat]
+
+Visual Studio Code extension to format and lint markdown files with remark.
+
+## Contents
+
+*   [What is this?](#what-is-this)
+*   [When should I use this?](#when-should-i-use-this)
+*   [Install](#install)
+*   [Use](#use)
+*   [Formatting](#formatting)
+*   [Plugins](#plugins)
+*   [Syntax](#syntax)
+*   [Compatibility](#compatibility)
+*   [Security](#security)
+*   [Contribute](#contribute)
+*   [License](#license)
+
+## What is this?
+
+This project is a Visual Studio Code (VS Code) extension that you can use in
+your editor to inspect and change markdown files.
+This extension is built around remark, which is a very popular ecosystem of
+plugins that work with markdown.
+You can choose from the 150+ plugins that already exist or make your own.
+
+See [remark][] for info on what the remark ecosystem is.
+
+## When should I use this?
+
+You can use this extension if you want to check (lint) and format markdown
+files from within your editor.
+
+To configure this extension, you define your preferred markdown style in a
+configuration file (`.remarkrc`, `.remarkrc.js`, etc. or in `package.json`).
+This file is picked up by `vscode-remark` and other tools (useful for
+contributors that don’t use VS Code).
+
+The configuration file is also used by [`remark-cli`][remark-cli], which is
+recommended to be used alongside `vscode-remark`, as an npm script and/or in
+CI, to enforce the markdown style.
+
+## Install
+
+[Get it from the VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=xxxx).
+Install through VS Code extensions.
+Search for `remark`.
+
+It can also be installed in VS Code: Launch VS Code Quick Open
+(<kbd>Ctrl</kbd> + <kbd>P</kbd>), paste the following command, and press enter.
+
+```txt
+ext install unifiedjs.vscode-remark
+```
+
+See the
+[extension installation guide](https://code.visualstudio.com/docs/editor/extension-gallery)
+for details.
+
+## Use
+
+This example enables VS Code users that have this extension installed to see
```

Comment:
Yes, I like that. I did keep the markdown example, because it’s very likely to actually show problems, whereas opening a file in a project where `remark-cli` is configured probably adheres to those rules.